libs/deps/mas/merch-card-collection-header.js

```diff
@@ -1,4 +1,8 @@
+<<<<<<< HEAD
```

Comment:
merge issue?
not sure if this file was deleted recently, could you try to re-install and re build /libs/features/mas?